lblod/app-subsidiedatabank
# [FEATURE] intital version

## ID
  DL-5280

 ## Description

This PR adds all elements needed for filtering in the frontend. We used the search-query-management service and ember-submission-form-fields to generate the filtering form. 

 ## Type of change


 ## Related services

 - [search-query-management-service](https://github.com/lblod/toezicht-search-query-management-service)

 ## How to test

This PR should be tested together with the following frontend PR:
- https://github.com/lblod/frontend-subsidiedatabank/pull/2

To test:
- setup app and frontend on correct branches
- wait for migrations to run 
- wait for delta-consumer to consume all data from loket
- mock login to frontend
- test all filters and see if they work

 ## Links to other PR's

 - https://github.com/lblod/frontend-subsidiedatabank/pull/2
